src/libraries/System.Runtime/tests/System.Globalization.Extensions.Tests/NlsTests/System.Globalization.Extensions.Nls.Tests.csproj

```diff
@@ -17,6 +17,8 @@
              Link="IdnMapping\Data\Unicode_11_0\Unicode_11_0_IdnaTest.cs" />
     <Compile Include="..\IdnMapping\Data\Unicode_13_0\Unicode_13_0_IdnaTest.cs"
              Link="IdnMapping\Data\Unicode_13_0\Unicode_13_0_IdnaTest.cs" />
+    <Compile Include="..\IdnMapping\Data\Unicode_13_0\Unicode_15_1_IdnaTest.cs"
```

Comment:
>13 [](http://example.com/codeflow?start=49&length=2)

15?

Reply:
Should be fixed now. Any idea how I can exercise this test suite?

Reply:
Need to manually run the test on a machine/container having ICU 74. I thought you already did that.

Reply:
Yes. Here's how I did it:

```
$ cd runtime
$ ./build.sh
$ cd src/libraries/System.Runtime/tests/System.Globalization.Extensions.Tests/
$ ../../../../../dotnet.sh build /t:Test
```

That failed before the changes in the PR and passed after.  However, that didn't fail even with the typo in this file, so I am wondering if there's more specific I need to do validate NlsTests and/or changes to this file.

Reply:
This test is not using ICU at all. It runs on NLS. It looks, NLS results are resilient to both v13 and v15. The original failure we are trying to fix is specific when using ICU. As long as the test pass when using ICU, that will be good enough.

Reply:
Curious, are you running on Windows?

Reply:
No, only Fedora 40 here.

Reply:
Ok, that explain it :-) NlsTests are Windows only tests that will not run on Fedora.